airiv/README.md

```diff
@@ -0,0 +1,10 @@
+# airiv
+**AIRiV: AI RIsk Vocabulary**
+
+**Contact:**
+This ontology is created and maintained by:
+Delaram Golpayegani
+PhD Researcher
+ADAPT Centre, Trinity College Dublin, Dublin, Ireland
+delaram.golpayegani@adaptcentre.ie
+GitHub: delaramglp
```

Comment:
To preserve the apparently desired linebreaks, and to enliven the GitHub handle --
```suggestion
**Contact:**  
This ontology is created and maintained by:  
Delaram Golpayegani  
PhD Researcher  
ADAPT Centre, Trinity College Dublin, Dublin, Ireland  
delaram.golpayegani@adaptcentre.ie  
GitHub: @delaramglp
```